seml/queue_experiments.py

```diff
@@ -256,14 +264,35 @@ def queue_configs(collection, seml_config, slurm_config, configs):
                  'config_hash': make_hash(c),
                  'queue_time': datetime.datetime.utcnow()}
                 for ix, c in enumerate(configs)]
+
     collection.insert_many(db_dicts)
 
 
+def get_imported_files(executable, root_dir):
+    exe_path = os.path.abspath(executable)
+    sys.path.insert(0, os.path.dirname(exe_path))
+    exp_module = importlib.import_module(os.path.splitext(os.path.basename(executable))[0])
```

Comment:
I've introduced a [new import_exe function](https://github.com/TUM-DAML/seml/pull/13/commits/922ae25e02c9129526d637743258c284295cb5a8#diff-f801f5c58e30a9deb5bde372f355e521R268) today. Use that instead.

Reply:
I agree. I think the cleanest would be to first merge your branch into master and then merge master into this branch and performing that update. Or do you have another suggestion?

Reply:
Yes, I agree.

Reply:
(Probably best do a rebase, not a merge, though.)

Reply:
Ok, I've done the rebase.